staging/src/k8s.io/client-go/dynamic/simple.go

```diff
@@ -366,6 +396,17 @@ func (c *dynamicResourceClient) ApplyStatus(ctx context.Context, name string, ob
 	return c.Apply(ctx, name, obj, opts, "status")
 }
 
+// takes an empty name for endpoints that don't use a name.
+func (c *dynamicResourceClient) validateURLSegments(name string) error {
+	if msgs := rest.IsValidPathSegmentPrefix(c.namespace); len(msgs) != 0 {
```

Comment:
I see `IsValidPathSegmentName` in rest package also. Why is the "Prefix" variant correct here?

https://github.com/kubernetes/kubernetes/blob/81027fd4247ad11f4473962b50fb21a7cf3caa85/staging/src/k8s.io/client-go/rest/request.go#L1369-L1385

Reply:
That's totally a mistake, I never meant to call "...Prefix" but "...Name".